README.md

```diff
@@ -82,6 +82,61 @@ This should only happen in when multiple nodes are writing events as delayed eve
 order.
 
 
+## Deployment to EKS/GKE
+
+The Akka platform operator can be used to deploy this application to EKS for testing. 
+
+### Deploying to a new cluster with terraform
+
+Configure you aws client `aws configure`
+
+```
+cd terraform
```

Comment:
I did this with terraform as there are quite a lot of manual steps for EKS/RDS and i want to be able to tear this down over the weekend